src/kafka_producer.rs

```diff
@@ -3,24 +3,21 @@ use kafka::producer::{Producer, Record, RequiredAcks};
 use rdev::Key;
 use std::time::Duration;
 
-pub fn send_to_kafka(msg: Key) {
+pub fn send_to_kafka(key_msg: Key) {
     // env_logger::init();
     let broker = "localhost:9092";
     let topic = "test";
 
-    let data = serde_json::to_string(&msg).unwrap();
+    let key_data = serde_json::to_string(&key_msg).unwrap();
+    let msg: String = key_data + " " + &chrono::offset::Local::now().to_string();
```

Comment:
This should not be here, current timestamp should be calculated before when the key is released (might as well consider time b/w press and release if that matters --- @imsrbh - take note please).

I'll work on moving this to a more appropriate position later.

Reply:
Okay!!